src/components/CardDetailsPage.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div id="card_details_page">
+      <div class="row justify-content-between mb-5 mr-5">
+        <logo :company_name="company_name" :app_name="app_name" style="padding-left:30px;margin-left: 35px"/>
+        <div class="glyphicon text-right align-self-center">
+          <a href="#" @click="getSearchPage">X</a>
+        </div>
+      </div>
+      <film-detail-card :imageUrl="card.imageUrl" :title="card.title" :releaseDate="card.releaseDate" :duration="card.duration" :genre="card.genre" :description="card.description" :rate="card.rate" style="margin:35px 35px;"/>
+    <div class="result_control">
+      <div class="bold_span col-4 text-center">
+        <span>Films by {{card.genre}} genre</span>
+      </div>
+    </div>
+    <film-gallery :cards="cards_by_genre" />
+    <custom-footer :company_name="company_name" :app_name="app_name" />
+  </div>    
+</template>
+
+<script>
+import "../assets/cardDetailsPage.css";
+import CustomFooter from './CustomFooter.vue';
+import FilmDetailCard from './FilmDetailCard.vue';
+import FilmGallery from './FilmGallery.vue';
+import Logo from './Logo.vue';
+import { EventBus } from "../event-bus";
+
+export default {
+  components: { Logo, FilmDetailCard, FilmGallery, CustomFooter },
+  props: {
+    card: Object
+  },
+  data() {
+    return {
+      company_name: "netflix",
```

Comment:
Consider keeping such constants under "src/core/constants" folder that you create after one of the above comments